### ISSUE TITLE
Weird behavior of measurement extraction after deleting custom measurements.

### ISSUE THREAD

**Issue by dimethylglymaxim** [SchmollerLab/Cell_ACDC#263]
### Description:

I tried to test out the custom measurements feature. It worked, but after I deleted the custom measurement in the measurements window, I'm constantly getting errors.

The first one is when I try to save the file with measurements (terminal output 1);
The second one is when I try to combine measurements from different positions (terminal output 2).

### Way to reproduce:

1. Create custom measurement;
2. Delete custom measurement;
3. Enjoy.

### Terminal output:

```Shell
Terminal output 1:

==============================
Traceback (most recent call last):
  File "C:\Users\myermolaie\Miniconda3\envs\acdc\lib\site-packages\pandas\core\computation\scope.py", line 198, in resolve
    return self.resolvers[key]
  File "C:\Users\myermolaie\Miniconda3\envs\acdc\lib\collections\__init__.py", line 941, in __getitem__
    return self.__missing__(key)            # support subclasses that define __missing__
  File "C:\Users\myermolaie\Miniconda3\envs\acdc\lib\collections\__init__.py", line 933, in __missing__
    raise KeyError(key)
KeyError: 'GFP_mean'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\myermolaie\Miniconda3\envs\acdc\lib\site-packages\pandas\core\computation\scope.py", line 209, in resolve
    return self.temps[key]
KeyError: 'GFP_mean'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\myermolaie\Miniconda3\envs\acdc\lib\site-packages\cellacdc\gui.py", line 483, in _dfEvalEquation
    df[newColName] = df.eval(expr)
  File "C:\Users\myermolaie\Miniconda3\envs\acdc\lib\site-packages\pandas\util\_decorators.py", line 331, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\myermolaie\Miniconda3\envs\acdc\lib\site-packages\pandas\core\frame.py", line 4609, in eval
    return _eval(expr, inplace=inplace, **kwargs)
  File "C:\Users\myermolaie\Miniconda3\envs\acdc\lib\site-packages\pandas\core\computation\eval.py", line 353, in eval
    parsed_expr = Expr(expr, engine=engine, parser=parser, env=env)
  File "C:\Users\myermolaie\Miniconda3\envs\acdc\lib\site-packages\pandas\core\computation\expr.py", line 813, in __init__
    self.terms = self.parse()
  File "C:\Users\myermolaie\Miniconda3\envs\acdc\lib\site-packages\pandas\core\computation\expr.py", line 832, in parse
    return self._visitor.visit(self.expr)
  File "C:\Users\myermolaie\Miniconda3\envs\acdc\lib\site-packages\pandas\core\computation\expr.py", line 415, in visit
    return visitor(node, **kwargs)
  File "C:\Users\myermolaie\Miniconda3\envs\acdc\lib\site-packages\pandas\core\computation\expr.py", line 421, in visit_Module
    return self.visit(expr, **kwargs)
  File "C:\Users\myermolaie\Miniconda3\envs\acdc\lib\site-packages\pandas\core\computation\expr.py", line 415, in visit
    return visitor(node, **kwargs)
  File "C:\Users\myermolaie\Miniconda3\envs\acdc\lib\site-packages\pandas\core\computation\expr.py", line 424, in visit_Expr
    return self.visit(node.value, **kwargs)
  File "C:\Users\myermolaie\Miniconda3\envs\acdc\lib\site-packages\pandas\core\computation\expr.py", line 415, in visit
    return visitor(node, **kwargs)
  File "C:\Users\myermolaie\Miniconda3\envs\acdc\lib\site-packages\pandas\core\computation\expr.py", line 536, in visit_BinOp
    op, op_class, left, right = self._maybe_transform_eq_ne(node)
  File "C:\Users\myermolaie\Miniconda3\envs\acdc\lib\site-packages\pandas\core\computation\expr.py", line 456, in _maybe_transform_eq_ne
    left = self.visit(node.left, side="left")
  File "C:\Users\myermolaie\Miniconda3\envs\acdc\lib\site-packages\pandas\core\computation\expr.py", line 415, in visit
    return visitor(node, **kwargs)
  File "C:\Users\myermolaie\Miniconda3\envs\acdc\lib\site-packages\pandas\core\computation\expr.py", line 549, in visit_Name
    return self.term_type(node.id, self.env, **kwargs)
  File "C:\Users\myermolaie\Miniconda3\envs\acdc\lib\site-packages\pandas\core\computation\ops.py", line 85, in __init__
    self._value = self._resolve_name()
  File "C:\Users\myermolaie\Miniconda3\envs\acdc\lib\site-packages\pandas\core\computation\ops.py", line 109, in _resolve_name
    res = self.env.resolve(local_name, is_local=is_local)
  File "C:\Users\myermolaie\Miniconda3\envs\acdc\lib\site-packages\pandas\core\computation\scope.py", line 211, in resolve
    raise UndefinedVariableError(key, is_local) from err
pandas.errors.UndefinedVariableError: name 'GFP_mean' is not defined
******************************

Terminal output 2:

Launching "Combine measurements from multiple tables" utility...
Asking to select experiment folders...
Processing experiment n. 1/1, Position_1 (1/17)
Traceback (most recent call last):
  File "C:\Users\myermolaie\Miniconda3\envs\acdc\lib\site-packages\cellacdc\workers.py", line 2150, in run
    result = self.run_iter_exp(exp_path, pos_foldernames, i, tot_exp)
  File "C:\Users\myermolaie\Miniconda3\envs\acdc\lib\site-packages\cellacdc\workers.py", line 2118, in run_iter_exp
    equationsConfig = self.addEquationsToConfigPars(
TypeError: addEquationsToConfigPars() takes 3 positional arguments but 4 were given
```

### Version information:

```Shell
>>> import sys; print(sys.version)
3.9.15 (main, Nov 24 2022, 14:39:17) [MSC v.1916 64 bit (AMD64)]
>>> import platform; print(platform.platform())
Windows-10-10.0.19045-SP0
>>> import cellacdc; print(f'Cell-ACDC version: {cellacdc.__version__}')
Cell-ACDC version: 1.2.4rc62
```

**Comment by ElpadoCan** [comment]
Hi Max,

I'm sorry you are having these issues. For the first terminal output, I agree that is annoying to see all those errors, but the process should have ended anyway without impacting the other measurements, is that correct? 

In any case, something went wrong when deleting the measurement and there is still some trace left, I will run some tests now. In the meantime, you should be able to get rid of it by manually deleting the `combine_metrics.ini` file (or deleting the specific equation inside the file, in any text editor) saved in your user folder inside the `acdc-metrics` folder. Something like `C:\Users\Frank\acdc-metrics\combine_metrics.ini`. This is the file that gets loaded with any dataset and if the evaluation of the combined metric for that dataset raises an exception the error is logged and the metric is skipped. 

For example, in your case, it complains that the column `GFP_mean` doesn't exist, probably because the dataset you loaded doesn't have the GFP channel. As I said, the metric was just skipped and the other measurements should be fine.

For the terminal output 2 instead, it's a critical bug that I will fix soon, I will keep you updated, thank you for your patience! 

Cheers,
Francesco

**Comment by dimethylglymaxim** [comment]
Hi Francesco,

You are right regarding my complaint number 1 - measurements were saved as expected. Also, I deleted combine_metrics.ini just as you suggested and it worked. Thank you very much!

Cheers, 
Max

**Comment by ElpadoCan** [comment]
Hi Max, I just released `v1.2.5` with a fix for your issue 2 and some improvements on error reporting for issue 1. It should now report just a warning with a short message instead of a full traceback. 

However, I could not replicate issue 1 because, in theory, it should have realised that GFP_mean was not available and it should have skipped it altogether (instead of trying and failing). 

Anyway, if it happens again, let me know if the error messages are shorter and look more like warnings. 

To update, activate the `acdc` environment and run the command `pip install -U cellacdc`. Let me know how it goes, thanks! 

Cheers,
Francesco
